README.md

```
@@ -38,27 +46,193 @@ func main() {
 Full documentation is available at [https://snider.github.io/Poindexter/](https://snider.github.io/Poindexter/)
 
+Explore runnable examples in the repository:
+- examples/dht_ping_1d
+- examples/kdtree_2d_ping_hop
+- examples/kdtree_3d_ping_hop_geo
+- examples/kdtree_4d_ping_hop_geo_score
+- examples/dht_helpers (convenience wrappers for common DHT schemas)
+- examples/wasm-browser (browser demo using the ESM loader)
+- examples/wasm-browser-ts (TypeScript + Vite local demo)
+
+### KDTree performance and notes
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)